package.json

```diff
@@ -92,7 +92,7 @@
     "react-addons-shallow-compare": "15.3.0",
     "react-addons-update": "15.3.0",
     "react-click-outside": "2.1.0",
-    "react-day-picker": "1.3.2",
+    "react-day-picker": "2.4.1",
```

Comment:
I think you should _shrinkwrapy_ to lock down our dependency versions.
Doc page -> https://github.com/Automattic/wp-calypso/blob/master/docs/shrinkwrap.md